app/controllers/api/v1/items/merchant_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::Items::MerchantController < ApplicationController
+
```

Comment:
felt this was the most restful solution, a merchant controller under the umbrella of items